board-viewer.html

```diff
@@ -270,10 +278,67 @@
 
   listeners: {
     "iron-resize" : "resize",
+    "touchmove": "_onPinch",
+    "wheel": "_onScroll",
     "board.response": "_boardLoaded",
     "board.error": "_boardLoadError"
   },
 
+  _onScroll: function(event) {
+    /* Inverted aim. */
+    if (event.deltaY < 0) {
+      this.zoom = Math.max(Math.min(this.zoom + 0.1, 2.0), 0);
```

Comment:
I did some test with this values most part of the time and I see a good operability, min-zoom as 0.5 does not look so well, I think the zoom should to return to original size (0), 2.0 is not so big, not so small, this is only my personal perception.